core/ide/src/main/java/org/alice/ide/member/MemberTabComposite.java

```diff
@@ -197,20 +199,34 @@ private List<MethodsSubComposite> getSubCompositesAlphabetically() {
     if (instanceFactory != null) {
       AbstractType<?, ?, ?> type = instanceFactory.getValueType();
       while (type != null) {
-        type.getDeclaredMethods();
-        for (AbstractMethod method : type.getDeclaredMethods()) {
-          if (this.isAcceptable(method)) {
-            if (isInclusionDesired(method)) {
-              methods.add(method);
+        if (type instanceof NamedUserType) {
+          NamedUserType namedUserType = (NamedUserType) type;
+
+          UserMethodsSubComposite userMethodsSubComposite = this.getUserMethodsSubComposite(namedUserType);
+          for (AbstractMethod method : userMethodsSubComposite.getMethods()) {
+            if (this.isAcceptable(method)) {
```

Comment:
Every use of `isAcceptable` is followed by `isInclusionDesired`. (But not the other way)
Not sure if having the former call the latter simplifies anything, since it would require adding `isAcceptable` or calls to super to two methods.